index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>My Portfolio</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <header>
+            <img class="logo" alt="Logo" src="./images/image_geometry_1.png">
+            <img class="menu" alt="Menu" src="./images/ic_menu.svg">
+        </header>
+        <div class="headline">
+            <div class="name">Emily Robertson</div>
+            <div class="intro">Hello! I am a software developer! 
+                I can help you build a product, 
+                feature or website. Take a look at my works.
+                If you like what you see and have a project 
+                you need coded, don’t hesitate and contact me.
+            </div>
```

Comment:
- [ ] Please use `hx (examples: <h1>,<h2>,<h3>, etc...)` for titles and headlines as it would more meaningful, i.e `<h1>hello...</h1> <h2>I am....</h2>`